development-vm/replication/common-args.sh

```diff
@@ -110,8 +110,11 @@ do
   esac
 done
 
-if [ $TOKEN_SET -eq 0 ]
+if [ $TOKEN_REQUIRED -eq 1 ]
   then
   usage
+  echo '---------------------------'
+  echo "!! MISSING MFA/2FA TOKEN !!"
```

Comment:
Maybe explain how to add it?

Reply:
It already is in the usage output, this is just to highlight what is missing.